_data/library/versions.yml

```diff
@@ -34,3 +34,4 @@
 - 1.13.0
 - 1.16.0
 - 1.17.0
+- 1.18.0
```

Comment:
Should this be `1.18.1`?

Reply:
No, this is the list of base versions for IR. They're all `.0`s.